post-scan-actions/aws-python-conformity-custom-check/README.md

```diff
@@ -4,10 +4,13 @@ After a scan occurs and a malicious file is detected, this example Lambda functi
 
 ## Prerequisites
 
-1. **Configure Conformity API Key**
-    - Log into Conformity and select your username and select `User settings`.
-    - Select `API Keys`.
-    - Select `+ New API Key`, then select `Generate Key`.
+1. **Configure Clond One API Key for Conformity**
+    - In the upper-right corner of the Trend Micro Cloud One console, select the account where you want to add an API key.
+    - Click the account name and then `Account Settings`.
+    - Click on `API Keys` from the left hand side navigation.
+    - Click on `New` to create a new API Key.
+    - Input the API Key Alias, Role, Language, and Timezone.
+    - Clicking on the `Next` button will generate a new API Key for you.
```

Comment:
- Click the **Next** button to generate the new API key.

Reply:
Please use "click" rather than "clicking on" to make it clear to the user (1) that the user is expected to do something and (2) what the thing to do is. The comment "clicking on" makes it sound like an option that the user has, not an expected action.